includes/class-shortcodes.php

```diff
@@ -450,14 +450,18 @@ private function process_category_atts( $atts, &$query_args ) {
 
 		$requested_categories = array();
 
-		if ( isset( $atts['category'] ) ) {
+		if ( $atts['category'] ) {
 			$requested_categories = array_merge( $requested_categories, explode( ',', $atts['category'] ) );
 		}
 
-		if ( isset( $atts['categories'] ) ) {
+		if ( $atts['categories'] ) {
```

Comment:
To avoid php warnings, this will need to be:
if ( isset( $atts['categories'] ) && $atts['categories'] ) {